webrtc/RTCPeerConnection-createOffer-offerToReceive.html

```diff
@@ -0,0 +1,85 @@
+<!doctype html>
+<meta charset=utf-8>
+<title>Test legacy offerToReceiveAudio/Video options</title>
```

Comment:
@alvestrand what are our conventions for adding author & help meta tags? I have noticed this convention sprinkled throughout the wpt repo (primarily within css subset). I actually wish there were more references especially to the respective spec.

### Examples
  - Spec `help` example: https://github.com/w3c/web-platform-tests/blob/56ea52b8668ceee63ec196c59dd11ad7797e4f0d/html/semantics/embedded-content/the-video-element/video_initially_paused-ref.html#L5
  - `author` example: https://github.com/w3c/web-platform-tests/blob/95eb7be62d2b5a3e9ef1165fd333d353fe3d4b30/html/rendering/bindings/the-button-element/button-type-menu-historical.html#L4

Reply:
No idea about the convention for meta tags, I've not used them; this should have recommendations at the WPT project level - @foolip may want to comment, but that shouldn't block this PR.

Reply:
>  that shouldn't block this PR @alvestrand 
Agreed

Reply:
You can add it if you find it useful. Myself, I never add author info because it's in git history, and add spec links only if the path and filename doesn't have all the info you need, which is often the case.